src/main/java/org/agbrothers/englishbot/constant/State.java

```diff
@@ -8,4 +8,5 @@
     public static final String LESSONS = "Lesson";
     public static final String DICTIONARY = "Dictionary";
     public static final String ADD_WORD_TO_DICTIONARY = "Додати нове слово";
+    public static final String DELETING_WORD = "Видалити слово";
```

Comment:
Could you please change the literals of these two constants to the English values?